lib/web_ui/lib/src/ui/canvas.dart

```diff
@@ -357,8 +357,8 @@ abstract class Canvas {
   ///
   /// Use [ClipOp.difference] to subtract the provided rectangle from the
   /// current clip.
-  void clipRect(Rect/*!*/ rect,
-      {ClipOp clipOp/*!*/ = ClipOp.intersect, bool/*!*/ doAntiAlias = true});
+  void clipRect(Rect rect,
+      {ClipOp clipOp/*!*/ = ClipOp.intersect, bool doAntiAlias = true});
```

Comment:
Leftover comment syntax?

Reply:
Good catch. The comment syntax was on the variable, not on the type, so the migration tool missed it. Fixed.